app/controllers/distributions_controller.rb

```diff
@@ -110,6 +110,16 @@ def update
     end
   end
 
+  def picked_up
+    if Distribution.find(params['id']).picked_up!
+      flash[:notice] = 'Distribution status updated!'
+    else
+      flash[:error] = "The status couldn't be updated"
```

Comment:
Our end users, who are mostly not tech-savvy, are the ones who will be seeing these flash notices. Could you make the flash notices be a little more contextualized? Something like: 
`This distribution has been marked as being picked up!` and `Sorry, we encountered an error when trying to mark this distribution as being picked up` -- something along those lines.

Thank you!